src/main/asciidoc/configuring-cors.adoc

```diff
@@ -0,0 +1,71 @@
+[[customizing-sdr.configuring-cors]]
+= Configuring CORS
+
+For security reasons, browsers prohibit AJAX calls to resources residing outside the current origin. When working with client-side HTTP requests issued by a browser you want to enable specific HTTP resources to be accessible.
+
+Spring Data REST supports as of 2.6 http://en.wikipedia.org/wiki/Cross-origin_resource_sharing[Cross-origin resource sharing] (CORS) through http://docs.spring.io/spring/docs/{version}/spring-framework-reference/html/cors.html[Spring's CORS] support.
+
+
+== Repository interface CORS configuration
+
+You can add a `@CrossOrigin` annotation to your repository interfaces to enable CORS for the whole repository. By default `@CrossOrigin` allows origins and `HEAD` and `GET` HTTP methods:
```

Comment:
`GET` and `HEAD` are in sync with Spring MVC (initially it's the same behavior) and to not allow any methods that change data by default. 

We check the allowed methods on collection/item level inside the particular controllers – we _could_ derive the allowed methods from there but the CORS configuration would be required to mirror the logic that is already present inside the controllers.